lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -354,6 +358,28 @@ FUNCTIONKIND_VALUES(VALUE)
         case FunctionKind::Array_indexOf:
             library->AddMember(scriptContext->GetLibrary()->GetEngineInterfaceObject()->GetCommonNativeInterfaces(), PropertyIds::builtInJavascriptArrayEntryIndexOf, func);
             break;
+        case FunctionKind::Array_some:
+            library->AddMember(scriptContext->GetLibrary()->GetEngineInterfaceObject()->GetCommonNativeInterfaces(), PropertyIds::builtInJavascriptArrayEntrySome, func);
+            break;
+        case FunctionKind::Array_every:
+            library->AddMember(scriptContext->GetLibrary()->GetEngineInterfaceObject()->GetCommonNativeInterfaces(), PropertyIds::builtInJavascriptArrayEntryEvery, func);
+            break;
+        case FunctionKind::Array_includes:
+            library->AddMember(scriptContext->GetLibrary()->GetEngineInterfaceObject()->GetCommonNativeInterfaces(), PropertyIds::builtInJavascriptArrayEntryIncludes, func);
+            break;
+        case FunctionKind::Array_reduce:
+            library->AddMember(scriptContext->GetLibrary()->GetEngineInterfaceObject()->GetCommonNativeInterfaces(), PropertyIds::builtInJavascriptArrayEntryReduce, func);
+            break;
+        case FunctionKind::Array_flat:
```

Comment:
Why does `Array_flat` match to `builtInJavascriptArrayEntryMap`?

Reply:
That is actually incorrect, I removed it 7dcbdc4ab8f74087ae5b3df6fe6fe99fb1b648e7.